lib/task-data/tasks/catalog-perccli.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Catalog perccli',
+    injectableName: 'Task.Catalog.perccli',
+    implementsTask: 'Task.Base.Linux.Catalog',
+    options: {
+        adapter: '0',
```

Comment:
For a catalog job, it better to scan all adapters and record all adapters' catalog data, is there any difficult to implement like this?